src/MegaPoker.sol

```diff
@@ -34,6 +34,7 @@ contract PokingAddresses {
     address constant univ2wbtcdai   = 0x5bB72127a196392cf4aC00Cf57aB278394d24e55;
     address constant matic          = 0x8874964279302e6d4e523Fb1789981C39a1034Ba;
     address constant wsteth         = 0xFe7a2aC0B945f12089aEEB6eCebf4F384D9f043F;
+    address constant crvv1ethsteth  = 0x2b465de3e69a2EC00158F0E3B4614e3582430ab2;
```

Comment:
This has been redeployed: `0x0A7DA4e31582a2fB4FD4067943e88f127F70ab39 `

Reply:
Done